routes/nodeMailer.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const transporter = require('../transporter');
+
+const emailRouter = express();
+
+emailRouter.use(express.json());
+
+emailRouter.post('/send', (req, res) => {
+  const { email, messageHtml } = req.body;
+  const mail = {
+    from: `Bob Ross bobross69pogchamp@gmail.com`,
```

Comment:
Can you change the email in the '"from" field to be the email from the credentials field in the .env file, so it would be consistent when a different email is used

Reply:
done!